### PR TITLE
ratik: make config style consistent with fenwick

### PR DESCRIPTION
**Pull request by pdowler**
add info log messages when queries return iterators
add summary log message at end of validation

### CODE REVIEW

File: ratik/README.md [hunk 1]
```diff
@@ -18,19 +18,30 @@ org.opencadc.ratik.inventory.username={dbuser}
...
+
 # remote inventory query service (luskan)
 org.opencadc.ratik.queryService={resourceID of remote TAP service with inventory data model}
 
 # artifact uri bucket filter
 org.opencadc.ratik.buckets={uriBucket prefix or range of prefixes}
 
 # selectivity
-org.opencadc.inventory.util.ArtifactSelector={fully qualified classname of ArtifactSelector implementation}
+org.opencadc.ratik.artifactSelector={all|filter}
```
Comment (jburke-cadc) [inline ratik/README.md:32]: The org.opencadc.ratik.artifactSelector property name in the example below for artifact-filter.sql needs to be updated.

File: ratik/README.md [hunk 2]
```diff
@@ -18,19 +18,30 @@ org.opencadc.ratik.inventory.username={dbuser}
...
 
-# local site type, true = global site, false = storage site
-org.opencadc.ratik.trackSiteLocations={true|false}
+# artifact uri bucket filter
+org.opencadc.ratik.buckets={uriBucket prefix or range of prefixes}
 ```
```
Comment (jburke-cadc) [inline ratik/README.md:35]: org.opencadc.ratik.buckets is listed twice in the example properties.

File: ratik/src/main/java/org/opencadc/ratik/ArtifactValidator.java [hunk 3]
```diff
@@ -394,16 +394,16 @@ protected void validateRemote(Artifact remote)
         log.debug("checking explanation 3");
         if (this.remoteSite == null) {
             try {
-                log.debug("starting transaction");
-                this.transactionManager.startTransaction();
-                log.debug("start txn: OK");
+                //log.debug("starting transaction");
+                //this.transactionManager.startTransaction();
```
Comment (jburke-cadc) [inline ratik/src/main/java/org/opencadc/ratik/ArtifactValidator.java:398]: The start and commit of the transaction are commented out?

Comment (pdowler) [reply ratik/src/main/java/org/opencadc/ratik/ArtifactValidator.java:398]: There is only one action so do not need explicit txn.

**Pull request merged**